template/template.go

```diff
@@ -42,14 +42,22 @@ type Template struct {
 	ExternalURL *url.URL
 }
 
+// Option is generic modifier of the text and html templates used by a Template
```

Comment:
```suggestion
// Option is generic modifier of the text and html templates used by a Template.
```

Reply:
Prometheus-based projects / Mimir comments must end with a period and are capitalized.

Reply:
Ugh, I remember implementing the check in Thanos CI way back :sweat_smile:
 thanks for pointing it out.